yahoo.py

```diff
@@ -1,388 +1 @@
-import pandas as pd
-import numpy as np
-from bs4 import BeautifulSoup
 import requests
-import random
-import io
-import re
-from datetime import datetime, date, timedelta
-
-class ticker:
-    def __init__(self, ticker):
```

Comment:
Undo this change. Restore all content of yahoo.py . Just do necessary changes for this task